src/engine.cpp

```diff
@@ -141,14 +141,19 @@ void Engine::verify_networks() const {
 }
 
 void Engine::load_networks() {
-    networks.big.load(binaryDirectory, options["EvalFile"]);
-    networks.small.load(binaryDirectory, options["EvalFileSmall"]);
+    load_big_network(options["EvalFile"]);
+    load_small_network(options["EvalFileSmall"]);
+    threads.clear();
```

Comment:
isn't this redundant since the two func calls do that already?

Reply:
ah indeed :D